index.js

```diff
@@ -2,18 +2,14 @@ require('dotenv').config()
 
 const Discord = require('discord.js')
 const nemojis = require('node-emoji')
-const get = require('get-value')
```

Comment:
Would need to also remove this from `package.json` by running `npm uninstall get-value`, otherwise the dependency will still be included.